SegmentEditorFloodFilling/SegmentEditorFloodFilling.py

```diff
@@ -107,7 +107,7 @@ def test_SegmentEditorFloodFilling1(self):
     slicer.mrmlScene.AddNode(segmentEditorNode)
     segmentEditorWidget.setMRMLSegmentEditorNode(segmentEditorNode)
     segmentEditorWidget.setSegmentationNode(segmentationNode)
-    segmentEditorWidget.setMasterVolumeNode(masterVolumeNode)
+    segmentEditorWidget.SetSourceVolumeNode(sourceVolumeNode)
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.